### PR TITLE
Improved release page build method

### PR DESCRIPTION
**Pull request by RKuttruff**
~~Does NOT make these changes live yet~~

- Updates website downloads page
  - Added table for release history and a table for latest SDAP components
  - Fixed ASF compliance issue for signature and checksum links
- Built changes for SDAP 1.1.0 release on 2023-05-16 and staged changes to go live on PR merge

### CODE REVIEW

File: source/downloads.html [hunk 1]
```diff
@@ -8,21 +8,80 @@ <h1>Download Apache SDAP (incubating) releases</h1>
...
+    {% for component in latest %}
+    <tr>
+        <td>{{ component.displayName }}</td>
+        <td>{{ component.release }}</td>
+        <td>{{ component.releaseDate }}</td>
+        <td>
+            <a href="https://www.apache.org/dyn/closer.lua/incubator/sdap/{{ component.release }}-incubating/apache-sdap-{{ component.component }}-{{ component.release }}-incubating-src.tar.gz">
+                [source] apache-sdap-{{ component.component }}-{{ component.release }}-incubating-src.tar.gz
+            </a>
+            <a href="https://www.apache.org/dyn/closer.lua/incubator/sdap/{{ component.release }}-incubating/apache-sdap-{{ component.component }}-{{ component.release }}-incubating-src.tar.gz.asc">
```
Comment (RKuttruff) [inline source/downloads.html]: Per [CalvinKirs](https://lists.apache.org/thread/r1om26omngvqjz1p2c763ttt3mndmmpk), these links should be to `downloads.apache.org` and not the mirror selector

Comment (RKuttruff) [reply source/downloads.html]: Ex: `https://downloads.apache.org/incubator/sdap/1.0.0-incubating/apache-sdap-ingester-1.0.0-incubating-src.tar.gz.asc`

Comment (RKuttruff) [reply source/downloads.html]: I marked the wrong lines. Should just be the .asc and .sha512 links that are updated. The .tar.gz links are fine

Comment (RKuttruff) [reply source/downloads.html]: Done

**Review by ngachung — APPROVED** [review]

**Comment by ngachung** [comment]
Thank you, @RKuttruff !

**Pull request merged**